### DISCUSSION TITLE
Switch Kernel?

### DISCUSSION THREAD

**Discussion by fsociety3765** [yuk7/ArchWSL#250] [Q&A] [answered]
Hi all,

Just got ArchWSL setup and running. One thing I noticed after installing neofetch was the kernel is 4.4.0-19041-Microsoft. I also have an Ubuntu WSL instance and the kernel on this is 5.10.16.3-microsoft-standard-WSL2. 

Is there a way to change this? 

Thanks,
FS

**Comment by yuk7** [comment, marked as answer]
Hi @fsociety3765 
Please upgrade your instance to WSL2.
Please run this command in Command Prompt
```
wsl --set-version Arch 2
```

**Comment by fsociety3765** [comment]
Thank you. 

I felt sure it was already on version 2 before. I feel certain I checked this. Bit upon checking just now, yes it is on version 1.
